core/kazoo_amqp/src/kz_amqp_history.erl

```diff
@@ -107,7 +107,7 @@ get(Consumer) ->
 
 -spec add_exchange(kz_amqp_exchange()) -> 'ok'.
 add_exchange(#'exchange.declare'{}=Exchange) ->
-    gen_server:call(?SERVER, {'add_exchange', Exchange}).
+    gen_server:call(?SERVER, {'add_exchange', Exchange}, 'infinity').
```

Comment:
Can we make this `?MILLISECONDS_IN_DAY` instead?

Reply:
fixing